Source/Tech Tree/Parts Browser/data/Engine_Config.json

```diff
@@ -4980,7 +4980,7 @@
         "spacecraft": "",
         "engine_config": "Agena",
         "upgrade": false,
-        "entry_cost_mods": "Model117ECM",
+        "entry_cost_mods": "7000, Model117ECM",
```

Comment:
Shouldn't that number go into Model117ECM instead?

Reply:
Maybe? The end result is the same, bit I can change it

Reply:
Is this resolved?

Reply:
I think it is fine as it